### ISSUE TITLE
spark 3

### ISSUE THREAD

**Issue by corymickelson** [goibibo/dataplatform_utils#26]
Hello, first thank you for providing this utility library, the delta to glue function is extremely useful. I see you have a spark 3 branch that's gone stale. Do you have plans to pick this back up? Or is there anything that documents the remaining work to be done so I can contribute?
Thanks

**Comment by sa1** [comment]
Yeah, I have plans to pick this up. However pull requests are welcome!
